### PR TITLE
55 generate weekly action items

### LINKED ISSUE #55: Generate `Weekly action items`
## Description
Splited task from #15, Generate the action items in the backend.
Depends on #61 
Creating action items will be scheduled once a week using Airflow in a day the client choose 

## DoD

- [ ] The weekly actions items are generated and ready for the frontend to present.
- [ ] Action items are implemented in a different service

### PR DESCRIPTION
**Pull request by chenyair**
closes #55

### CODE REVIEW

File: action-items-generator/src/app.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import cors from 'cors';
+import express, { Express } from 'express';
+import dotenv from 'dotenv';
+import createLogger from './utils/logger';
+import initDB from './db';
+import initConsumer from './consumer';
+import healthRoute from './routes/health.route';
+import 'express-async-errors';
+
+const logger = createLogger('app');
```
Comment (liadyacobi) [inline action-items-generator/src/app.ts]: maybe make this logger's name `action-items-generator`

File: shared-types/db.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+export const ReviewMongoseSchema = {
```
Comment (liadyacobi) [inline shared-types/db.ts]: fix typo mongose to mongoose

File: action-items-generator/src/app.ts (hunk 1 above)
Comment (yoavrichter30) [reply action-items-generator/src/app.ts]: Noder

File: action-items-generator/src/consumer.ts [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        const prefix = `${topic}[${partition} | ${message.offset}] / ${message.timestamp}`;
+        this.logger.info(`- ${prefix} ${message.key}#${message.value}`);
+
+        const request = IWeeklyActionItemsRequestSchema.parse(JSON.parse(message.value!.toString()));
+        const requestDate = new Date(request.date);
+        const weekBeforeRequest = new Date(requestDate.getTime() - 7 * 24 * 60 * 60 * 1000);
+
+        const lastWeekReviews = await reviewModel
+          .find({
+            businessId: request.client.businessId,
```
Comment (yoavrichter30) [inline action-items-generator/src/consumer.ts:39]: isnt it req.user in the other backends?

**Pull request merged**